doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -27,6 +27,13 @@ HOST: http://orion.lab.fiware.org/
 
 FIWARE-NGSI Simple API.
 
+This is a work in progress and is changing on a daily basis.
+Please send your comments to fiware-ngsi@lists.fiware.org
+
+In addition, note that a list of currently open discussions is available at
+https://github.com/telefonicaid/fiware-orion/issues/1022
+
+
```

Comment:
we need to add here the licensing schema of this specification, probably it would be CC or something like that